.github/workflows/event-merge-to-queue.yml

```diff
@@ -67,23 +122,22 @@ jobs:
 
   sanitize:
     needs: get-latest-redis-tag
-    secrets: inherit
     uses: ./.github/workflows/task-test.yml
+    secrets: inherit
     with:
+      env: ubuntu-latest
+      san: address
       get-redis: ${{ needs.get-latest-redis-tag.outputs.tag }}
       rejson-branch: master
       test-config: '' # run all tests
-      san: address
-      env: ubuntu-latest
 
   pr-validation:
     needs:
       - get-latest-redis-tag
       - lint
-      - test-linux
-      - test-macos
-      - coverage
+      - unified-test-matrix
       - run-on-intel
+      - coverage
       - sanitize
```

Comment:
Can't we have `coverage` and `sanitize` as a special config entry in the matrix? It's a shame we don't get a single matrix in the end.
Same goes to `run-on-intel` (although it add another complexity level, so I guess we can keep it as an exception)